src/frontend/src/tests/utils/derived.utils.ts

```diff
@@ -0,0 +1,131 @@
+import { tokenWithFallbackAsIcToken } from '$icp/derived/ic-token.derived';
+import {
+	btcAddressMainnet,
+	btcAddressTestnet,
+	ethAddress,
+	ethAddressCertified,
+	ethAddressNotCertified,
+	ethAddressNotLoaded
+} from '$lib/derived/address.derived';
+import { authIdentity, authNotSignedIn, authSignedIn } from '$lib/derived/auth.derived';
+import { balance, balanceZero } from '$lib/derived/balances.derived';
+import { isBusy } from '$lib/derived/busy.derived';
+import { exchangeInitialized, exchanges } from '$lib/derived/exchange.derived';
+import { userHasPouhCredential } from '$lib/derived/has-pouh-credential';
+import { routeNetwork, routeToken } from '$lib/derived/nav.derived';
+import {
+	combinedDerivedSortedNetworkTokens,
+	combinedDerivedSortedNetworkTokensUi
+} from '$lib/derived/network-tokens.derived';
+import {
+	networkAddress,
+	networkBitcoin,
+	networkEthereum,
+	networkICP,
+	networkId,
+	pseudoNetworkChainFusion,
+	selectedNetwork
+} from '$lib/derived/network.derived';
+import { networks, networksMainnets, networksTestnets } from '$lib/derived/networks.derived';
+import { pageToken } from '$lib/derived/page-token.derived';
+import { hideZeroBalances, showZeroBalances, testnetsEnabled } from '$lib/derived/settings.derived';
+import { testnets } from '$lib/derived/testnets.derived';
+import {
+	tokenDecimals,
+	tokenId,
+	tokenStandard,
+	tokenSymbol,
+	tokenToggleable,
+	tokenWithFallback
+} from '$lib/derived/token.derived';
+import {
+	enabledErc20Tokens,
+	enabledIcTokens,
+	enabledMainnetTokensUsdBalancesPerNetwork,
+	enabledTokens,
+	tokens,
+	tokensToPin
+} from '$lib/derived/tokens.derived';
+import { tick } from 'svelte';
+import type { Unsubscriber } from 'svelte/store';
+import type { MockInstance } from 'vitest';
+
+const derivedList = {
```

Comment:
My editor complains if we do not define a type for this, that's why in my snippet I suggest `Readable<unknown>[]`. An object works too but, it needs to be typed on my hand.

Reply:
I included the type. But we don't we have a rule to enforce type definition for non-explicit objects?

Reply:
Webstorm did not complained on the object declaration but, on it's usage. All good, thx.